client/unit-tests/coretypes/test_filtering.py

```diff
@@ -76,7 +82,7 @@ def test__format_filter(geojson, polygon):
             Dataset.metadata["some_str"] == "foobar",
             Dataset.metadata["some_float"] >= 0.123,
             Dataset.metadata["some_datetime"] > datetime.timedelta(days=1),
-            Dataset.metadata["some_geospatial"].intersects(polygon),  # type: ignore
```

Comment:
you don't have to fix this schema issue in this PR, but can you add a ticket with context to the backlog so we don't forget about it?

Reply:
#605